clang/test/CXX/drs/dr16xx.cpp

```diff
@@ -42,6 +42,28 @@ namespace dr1611 { // dr1611: dup 1658
   C c;
 }
 
+namespace dr1626 { // dr1626: no open
+// FIXME: current consensus for CWG2335 is that the examples are well-formed.
```

Comment:
It is not clear from https://www.open-std.org/jtc1/sc22/wg21/docs/cwg_active.html#1626 what the intent for that is.

The minutes are very terse
```
- I agree with you, would be happy to mark as NAD
- Will mark as P1
```

Reply:
Intent expressed in P1787 in PR description seems rather clear to me, as well as its application to CWG1626, but I don't insist.

Reply:
We had an offline discussion, and concluded that this test should be removed.